deployments/liqo/subcharts/advertisementOperator/templates/adv-deploy.yaml

```diff
@@ -9,6 +9,98 @@ metadata:
     app: liqo.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: advertisement-operator-cluster-role
```

Comment:
Just a comment on the fly. What about something along the lines of `liqo-advertisement-operator` (or `liqo.io:advertisement-operator`) as a name? I feel it would simplify the management as all the liqo cluster roles (and cluster rolebindings if modified accordingly) would be grouped together. Looking at our cluster, it also seems to be the standard practice of most other projects.